.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,22 +2,16 @@
 name: Bug report
 about: Create a bug report
 title: ''
-labels: ''
+labels: bug
 assignees: ''
 
 ---
 
-### Description
+**Description:** A clear and concise description of what the bug is.
 
-<!--
-* Please share short description of the problem
-  -->
+**Expected behavior:** A description of what you expected to happen.
 
-### Details
+**Actual behavior:** A description of what is actually happening.
 
-<!--
-* Include the relevant yaml, platform, and dotnet versions in use
-* If an error occurred on a public action, please share a link
-* Include any error messages received in text (search does not work for images)
-* Was this a regression from previous behavior?
-  -->
+**Repro steps:**  
```

Comment:
I think we should add 3 more sections:
1) Task version - customer can use specific task version and their issue can be reproduced only there, it is better to know
2) Platform (Ubuntu, macOS, Windows)
2) Runner type (Hosted / Self-hosted)

Reply:
Done.